git-extra/getgit

```diff
@@ -248,7 +248,7 @@ echo "be removed from the context menu is available as $rm_ctxmenu "
 echo ""
 # Setting ENV
 echo -n "Setting env ..."
-userpath=$(reg query "HKEY_CURRENT_USER\Environment" //v Path 2>/dev/null | awk 'NR==3 {print $NF}')
+userpath=$(reg query "HKEY_CURRENT_USER\Environment" //v Path 2>/dev/null | tail -n +3 | awk '{$1=$2=""; print substr($0,3)}')
```

Comment:
@Morilli would you mind explaining this a bit more in the commit message? Feel free to follow [this guidance](https://github.blog/2022-06-30-write-better-commits-build-better-projects/):
![image](https://github.com/git-for-windows/build-extra/assets/127790/dadddc72-2486-4491-82c1-e0aa768408e4)

Reply:
Have force pushed with a full explanation.